mojave/items/misc/radios.dm

```diff
@@ -1,13 +1,50 @@
+/obj/item/radio/ms13
+	icon = 'mojave/icons/objects/hamradio.dmi'
+	name = "hand Radio"
+	icon_state = "handradio"
+	inhand_icon_state = "handradio_"
+	desc = "A basic handheld radio that recieves over a relatively long range, unfortunately this one can't broadcast."
+
+	w_class = WEIGHT_CLASS_SMALL
+	custom_materials = list(/datum/material/iron=75, /datum/material/glass=25)
+	radio_broadcast = FALSE
+
+/obj/item/radio/ms13/can_receive(freq, level, AIuser)
+	if(ishuman(src.loc))
+		var/mob/living/carbon/human/H = src.loc
+		if(H.is_holding(src))
+			return ..(freq, level)
+	else if(AIuser)
+		return ..(freq, level)
+	return FALSE
+
+/obj/item/radio/ms13/broadcast
+	icon = 'mojave/icons/objects/hamradio.dmi'
+	name = "broadcast hand radio"
+	icon_state = "handradio"
+	inhand_icon_state = "handradio_"
+	desc = "A rare handheld radio that can send as well as recieve signals. The poor quality of broadcasts makes it unpleasent to listen to, and doing so too often is a good way to get lynched."
+	radio_broadcast = RADIOSTATIC_HEAVY
+
+/obj/item/radio/ms13/broadcast/prewar
+	icon = 'mojave/icons/objects/hamradio.dmi'
+	name = "pre-War hand radio"
+	icon_state = "handradio"
+	inhand_icon_state = "handradio_"
+	desc = "The best a handheld gets, this extremely rare radio can broadcast at reasonably high quality while remaining lightweight and portable."
+	radio_broadcast = RADIOSTATIC_MEDIUM
+
 /obj/item/radio/ms13/ham
 	name = "ham radio"
-	desc = "An amateur radio setup. The sound quality could be better, but it's better than screaming into the horizon."
+	desc = "An amateur radio setup. The sound quality could be better, but it beats listening to brahmin all day."
 	icon = 'mojave/icons/objects/hamradio.dmi'
 	icon_state = "radio_on"
-	canhear_range = 1
+	canhear_range = 7
 	pixel_y = 5
 	freerange = TRUE
 	anonymize = TRUE
 	anchored = TRUE
+	radio_broadcast = FALSE
```

Comment:
I don't really think we intend to have a receive only ham radio, just seems kind of silly when the receive only handhelds will be somewhat common. This can probably be nixed in favor of just a standard ham radio and then the broadcast set.